backend/src/modules/accounts/repositories/batchAccountsByIds.ts

```diff
@@ -8,6 +8,8 @@ export const batchAccountsByIds = async (
   hiddeBalance = true,
   accounts: Collection = database.collection(config.collections.accounts)
 ) => {
+  console.log("batchAccountsByIds", ids);
```

Comment:
should this be here?

Reply:
Lol, I was using it to debug the dataloader